RULES.md

```diff
@@ -142,7 +142,7 @@ def get_batch_size(workload_name: str) -> int
 
 - Submitters define a specific batch size for each [workload](#workloads).
 - For example, in advance, they can determine the largest batch size without running out of memory for each workload.
-- For the [randomized workloads](#randomized-workloads), the `workload_name` of the closest public workload will be used in this function.
+- For the [held-out workloads](#randomized-workloads), submitters may provide a batch size once the submission code is frozen and the held-out workloads are sampled from the randomized workloads. By default, this function will use the `workload_name` of the fixed workload it is based on.
```

Comment:
Perhaps we should allow submitters to specify batch sizes for all possible held out workloads?